MoltenVKShaderConverter/MoltenVKGLSLToSPIRVConverter/GLSLToSPIRVConverter.cpp

```diff
@@ -125,8 +147,8 @@ bool GLSLToSPIRVConverter::validateSPIRV() {
 void GLSLToSPIRVConverter::logGLSL(const char* opDesc) {
 	_resultLog += opDesc;
 	_resultLog += " GLSL:\n";
-	_resultLog += _glsl;
-	_resultLog += "\nEnd GLSL\n\n";
+	for (const auto& glsl : _glsls) _resultLog += glsl + "\n";
```

Comment:
Can you just wrap this in `{}` to avoid the hanging-`for`, please?

Reply:
@billhollings  I wrapped a couple statements in brackets as you requested [here](https://github.com/w103csh/MoltenVK/commit/5c3b265888b02fd83dc81cfe9c322fab6295f654). I wasn't sure if you wanted the  statements inside the braces on a separate line or not. I saw some `if`'s in the function above with a similar style so I went with that. Let me know if its what you want.

Reply:
Yup. Looks great. And thanks for noticing and fixing the additional hanging-if statements.